test/question.test.js

```diff
@@ -1,67 +1,135 @@
 process.env.NODE_ENV = 'test';
 const chai = require('chai');
 const expect = chai.expect;
+const sample_data = require('./test.data.json');
 
 const supertest = require('supertest');
 const app = require('../index');
 
+//positive test for post question
 describe('Question Test', function () {
     describe('Positive Tests', function () {
         it('should successfully post a question', async function () {
             const response = await supertest(app)
                 .post('/api/v1/question')
-
+                .set({
+                    Authorization: sample_data.auth_token,
+                    Accept: 'application/json',
+                })
                 .send({
                     question_text: 'My first age',
                     user_id: '24',
                 })
-                .set('Accept', 'application/json')
                 .expect(200);
-
             const resp_data = response.body;
-            console.log(resp_data);
             expect(resp_data).to.be.an('object');
-            expect(resp_data).to.have.property('user');
-            expect(resp_data.user.id).to.be.an('number');
+            expect(resp_data).to.have.property('user_id');
+            expect(resp_data.user_id).to.be.an('number');
         });
     });
 
-    // describe('Question Test', function () {
-    //     describe('Positive Tests', function () {
-    //         it('should successfully post a question', async function () {
-    //             const response = await supertest(app)
-    //                 .post('/api/v1/question')
+    //positive test for delete question
+    describe('Positive Tests', function () {
```

Comment:
Since the next test case is still part of the Positive Test suites then no need to use another describe, so remove line:29-32

Reply:
Remove line 29-32, it is redundant